### PR TITLE
Fix import format for snowflake_database_grant

### PR DESCRIPTION
**Pull request by kanata2**
## Description
According to the following code, we must pass 4 or 5 pipe-delimited values in order to use `terraform import`, but the documentation says that you should pass 3 values.
Indeed, `terraform import snowflake_database_grant.xxx 'xxx|||USAGE|false' works correctly, so I fixed the documentation.

https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/a4240bb6bd399141f3614b3a5e1ff54476de70b4/pkg/resources/grant_helpers.go#L114-L116

### Current Behaviour

```
$ terraform import snowflake_database_grant.xxx 'xxx|||USAGE|false'
...
snowflake_database_grant.xxx: Refreshing state... [id=xxx|USAGE|false]

Error: 4 or 5 fields allowed
```

### CODE REVIEW

**Comment by kanata2** [comment]
Ah, I found same cases which should be fixed and will fix its later.

**Comment by kanata2** [comment]
Fixed in 8cdb7ca

**Comment by ryanking** [comment]
@kanata2 what version was this on? We had a regression in, I think v0.20, related to this which was later reverted.

**Comment by kanata2** [comment]
@ryanking I used v0.18.1, but main branch seems to have same problem.

**Comment by kanata2** [comment]
I tried `terraform import snowflake_database_grant.xxx 'xxx|USAGE|false'`again, but got the same result.

```
$ terraform import snowflake_database_grant.xxx 'xxx|USAGE|false'
snowflake_database_grant.xxx: Importing from ID "xxx|USAGE|false"...
snowflake_database_grant.xxx: Import prepared!
  Prepared snowflake_database_grant for import
snowflake_database_grant.xxx: Refreshing state... [id=xxx|USAGE|false]

Error: 4 or 5 fields allowed


$ terraform import snowflake_database_grant.xxx 'xxx|||USAGE|false'
snowflake_database_grant.xxx: Importing from ID "xxx|||USAGE|false"...
snowflake_database_grant.xxx: Import prepared!
  Prepared snowflake_database_grant for import
snowflake_database_grant.xxx: Refreshing state... [id=xxx|||USAGE|false]

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

File: docs/resources/database_grant.md [hunk 1]
```diff
@@ -42,6 +42,6 @@ resource snowflake_database_grant grant {
 Import is supported using the following syntax:
 
 ```shell
-# format is database name | privilege | true/false for with_grant_option
-terraform import snowflake_database_grant.example 'databaseName|USAGE|false'
+# format is database name | | | privilege | true/false for with_grant_option
+terraform import snowflake_database_grant.example 'databaseName|||USAGE|false'
```
Comment (ryanking) [inline docs/resources/database_grant.md:46]: You need to update this file - https://github.com/chanzuckerberg/terraform-provider-snowflake/blob/main/examples/resources/snowflake_database_grant/import.sh 

and then run `make docs`.

**Review by ryanking — CHANGES_REQUESTED** [review]
same for all the other files

File: docs/resources/database_grant.md (hunk 1 above)
Comment (kanata2) [reply docs/resources/database_grant.md:46]: Ah, sorry. I'll run later.

Comment (kanata2) [reply docs/resources/database_grant.md:46]: ran at c8f9e2e

**Review by alldoami — APPROVED** [review]

**Comment by alldoami** [comment]
@kanata2 can you update your branch with main?

**Pull request merged**